src/stubs/ezodf/__init__.pyi

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 eprbell
```

Comment:
Update copyright notice to be 2023 and use your name, since this is a new file you introduced.